<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountMapDialog</class>
 <widget class="QDialog" name="AccountMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="mapTypeLabel">
     <property name="text">
      <string>Map type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="accountMapTableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addRowBtn">
       <property name="text">
        <string>Add row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRowBtn">
       <property name="text">
        <string>Delete row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>AccountMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
